Newsletters: add 232 (2023-01-04)

harding opened the pull request:
- [x] Lede, releases/RCs, topic links @harding 
- [x] Bitcoin Core 26265 @glozow 
- [x] Bitcoin Core 21576 @bitschmidty

bitschmidty commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,200 @@
+      useful to anyone participating in the signet tests.
+
+    - *Full-RBF peering node:* Peter Todd [announced][todd rbf node] a
+      patch on top of Bitcoin Core 24.0.1 that sets a full-RBF service
```
Could link here as it could be interesting for readers to see the other service bits 

https://github.com/petertodd/bitcoin/commit/c15b8d70778238abfa751e4216a97140be6369af

bitschmidty commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,200 @@
+      not always favor full-RBF.  He suggests three possible scenarios
+      for consideration.
+
+        Towns also analyzes the phase, "Bitcoin is money for enemies"
```
I dont feel strongly on it, but this paragraph seems out of place in a typical Optech newsletter.

bitschmidty commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,200 @@
+- [Eclair #2482][] allows sending payments using [blinded routes][topic
+  rv routing], which are paths whose last several hops are chosen
+  by the receiver.  The receiver uses onion encryption to obfuscate the
+  hop details and provides the encrypted data to the spender, along with
```
```suggestion
  hop details and provide the encrypted data to the spender, along with
```

LarryRuane replied:
I'm not sure about your suggestion, @bitschmidty; I think the subject of the verb "provides" is the receiver (singular). Maybe it seems like "provide" is correct because "details" is plural?

LarryRuane commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,200 @@
+
+- [Bitcoin Core #26628][] begins rejecting RPC requests that include the
+  same parameter name multiple times.  Previously, the daemon treated a
+  request with repeated parameters as if it only had the last of the repeated
```
```suggestion
  request with repeated JSON keys as if it only had the last of the repeated
```

Xekyo reviewed (commented):
LGTM, I have no additional comments at this time

adamjonas commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,200 @@
+- [Rust Bitcoin #1467][] adds methods for calculating the size in
+  [weight units][] of transaction inputs and outputs.
+
+- [Rust Bitcoin #1330][] removes the `PackedLockTime` type, requiring
+  code instead use the almost-identical `absolute::LockTime` type.  A
```
I think if the code part of this sentence is dropped it reads easier

```suggestion
- [Rust Bitcoin #1330][] removes the `PackedLockTime` type, requiring
  code use of the almost-identical `absolute::LockTime` type.  A
```

harding replied:
Changed to "downstream code".  I want to communicate that this isn't just an internal change, but a change that anyone depending on Rust Bitcoin for locktimes may need to account for (although hopefully it's something that will be caught by the linker when they next upgrade their dependencies, so this is just advance notice).

glozow commented on _topics/en/async-payments.md:
```diff
@@ -0,0 +1,92 @@
+Traditional onchain Bitcoin payments are asynchronous (async) because
+the receiver can generate an output script (Bitcoin address) and give
+that address to the spender at any time, and then the spender can pay
+that address at any time---even at a time when the receiver is offline.
```
```suggestion
that address at any time---even when the receiver is offline.
```

glozow reviewed (commented):
Apologies for the delay on my section!

harding commented on _posts/en/newsletters/2023-01-04-newsletter.md:
```diff
@@ -0,0 +1,203 @@
+      incoming connections on an IPv4 address.  He [found][todd probe]
+      that about 17% relayed a full-RBF replacement: a transaction which
+      replaced a transaction that did not contain the [BIP125][]
+      nSequence signal.  This indicates those nodes were running with
```
(I really like this phrasing change, thanks!)

harding commented:
Made edits (thanks @bitschmidty @LarryRuane @adamjonas @Xekyo @glozow !), reviewed @bitschmidty and @glozow sections (thanks!), and added releases, RCs, and topics.  Also added an item about Bitcoin Knots's signing key compromise.  Thanks everyone and happy new year!

bitschmidty commented:
ACK 68737451117f7301e974f0edda395889df2c4e1a squashed

bitschmidty commented:
Happy New Year Optech contributors and thank you for helping with this first newsletter of the year @harding @glozow @Xekyo @adamjonas @LarryRuane
